packages/react-components/react-avatar/src/components/AvatarGroup/useAvatarGroupStyles.ts

```diff
@@ -80,6 +72,12 @@ const useOverflowButtonStyles = makeStyles({
     outlineStyle: 'solid',
   }),
 
+  // This custom focus indicator is required for the pie layout due to the clip-path applied to the root
+  pieFocusIndicator: createCustomFocusIndicatorStyle({
+    ...shorthands.border(tokens.strokeWidthThick, 'solid', tokens.colorStrokeFocus2),
```

Comment:
sounds good, I'll try it out and see which one works best. Thanks!

Reply:
@behowell  After trying it out, I will have to keep it as id. The reasoning is that I feel this way, it makes it simpler rather than adding the `::after` pseudo-element, which shouldn't have any problems due to the button being completely transparent. This styling is only applied to the pie layout therefore the others work as expected. Another reason is that the border is "inside" the circle since by using `clip-path` the outline won't show, this is done with `inline-flex` because it adds `box-sizing: border-box` by default. Overall, I think this is the cleanest way, but I don't mind adding the `::after` and size it like the root if preferred.

Reply:
An right, I had forgotten that this is an empty button, separate from the element that contains the pie slices, right? In that case, changing its border thickness won't affect anything, and this should be fine.

Reply:
Yup, it's a plain button with transparent styling.